src/Rule/Rule.php

```diff
@@ -117,13 +117,21 @@ public static function parse(ParserState $oParserState): Rule
      * @param string $sRule
      *
      * @return array<int, string>
+     * The first item is the innermost separator (or, put another way, the highest-precedence operator).
+     * The sequence continues to the outermost separator (or lowest-precedence operator).
```

Comment:
I'd prefer this text to be part of the general description of this method instead of the `@return` annotation as the description is what I'd read first (as a human).

Reply:
Moved, adding context.